doc/api/url.md

```diff
@@ -303,7 +303,19 @@ to percent-encode may vary somewhat from what the [`url.parse()`][] and
 
 * {string}
 
-Gets and sets the port portion of the URL.
+Gets and sets the port portion of the URL. It can be an empty string and in
+that case the port depends on the protocol/scheme. Here are the
+supported protocols:
```

Comment:
Optional nit: Remove the `Here are the supported protocols:` sentence and end the previous sentence with `:`